### PR TITLE
feat(ui-ux): update sui backing address

### PR DESCRIPTION
**Pull request by chloezxyy**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link:

<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

**Comment by fullstackninja864** [comment]
@chloezxyy can you update test case too `proof-of-backing.spec.ts`

File: cypress/fixtures/proof-of-backing/backingAddresses.ts [hunk 1]
```diff
@@ -31,6 +31,10 @@ export const cakeBackingAddresses = [
     token: "dDOGE",
     link: "https://dogechain.info/address/9uv4fqPjSYNVNvqzbuGUMACBw67qQcLTxg",
   },
+  {
+    token: "dSUI",
+    link: "https://suiexplorer.com/address/0xe319ee27de5d3cb4a2345c8b714a69d6710ca05395e3709983d886061b1b818d",
+  },
```
Comment (fullstackninja864) [inline cypress/fixtures/proof-of-backing/backingAddresses.ts:37]: I meant to say can you add all remaining token in e2e please

Comment (pierregee) [reply cypress/fixtures/proof-of-backing/backingAddresses.ts:37]: Merging this @chloezxyy but let's add all remaining e2e that were not added before

Comment (pierregee) [reply cypress/fixtures/proof-of-backing/backingAddresses.ts:37]: in another PR

**Pull request merged**